### PR TITLE
Create 2022-02-09-underhanded-solidity-contest-2022-announcement.md

### PR DESCRIPTION
**Pull request by franzihei**
Date will be adjusted before posting to be able to display the post when building it locally.

### CODE REVIEW

File: _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+The top 3 submissions of the Underhanded Solidity Contest will also be awarded points for the upcoming [Paradigm CTF 2022](https://ctf.paradigm.xyz/).
+Those points will be awarded as though the winners solved a bonus challenge ("pseudo-challenge").
+
+Furthermore, the three winners will be added to the Board of Fame.
+
+The winners and all qualified submissions will receive a custom Underhanded Solidity Contest NFT.
+
+## The Submission 📨
+
+Before you submitting your code ensure that:
```
Comment (chriseth) [inline _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md:74]:
```suggestion
Before submitting your code ensure that:
```

File: _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
++ The entirety of your submission is licensed under an open-source license.
++ In your submission you use Solidity v0.8.0 or newer.
++ You have carefully read the [Coding Brief](https://underhanded.soliditylang.org/#coding-brief).
+
+Please email your submissions before the deadline [2022-03-16] to ``sol_underhanded@ethereum.org``. Entries should consist of a ZIP file containing a README describing your submission and how it works, and one or more Solidity files.
```
Comment (chriseth) [inline _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md:79]: Does the date in brackets render correctly?

File: _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+The winners and all qualified submissions will receive a custom Underhanded Solidity Contest NFT.
+
+## The Submission 📨
+
+Before you submitting your code ensure that:
++ The entirety of your submission is licensed under an open-source license.
++ In your submission you use Solidity v0.8.0 or newer.
++ You have carefully read the [Coding Brief](https://underhanded.soliditylang.org/#coding-brief).
+
```
Comment (chriseth) [inline _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md:78]: Do we want to re-state that it should contain a readme and a separate spoiler / rugpull.txt?

Comment (franzihei) [reply _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md:78]: done

File: _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md (hunk 2 above)
Comment (franzihei) [reply _posts/2022-02-09-underhanded-solidity-contest-2022-announcement.md]: added time 11:59PM UTC

**Pull request merged**